experimental/PropertyDDS/packages/property-properties/src/properties/abstractStaticCollectionProperty.js

```diff
@@ -347,10 +347,11 @@ export class AbstractStaticCollectionProperty extends BaseProperty {
 	 * Given an object that mirrors a PSet Template, assigns the properties to the values
 	 * found in that object.
 	 * See {@link setValues}
-	 * @param {object} in_values - The object containing the nested values to assign
+	 * @param {Object | Array | string} in_values - The object containing the nested values to assign
 	 * @param {boolean} in_typed - Whether the values are typed/polymorphic.
 	 * @param {boolean} in_initial - Whether we are setting default/initial values
 	 * or if the function is called directly with the values to set.
+	 * @protected
```

Comment:
TypeScript 5 notices if the visibility of an override does not match the base class signature.  In this case, '_setValues' was protected in the base class, but implicitly public here.